src/config.py

```diff
@@ -9,17 +6,23 @@ class Config:
   Class for loading and accessing the config file.
   """
 
-  def __init__(self):
-    self._json = {}
+  def __init__(self, configs: list[dict[str, str | bool]]):
+    self._config = {}
 
-  def load(self, config_filepath: str):
-    if not os.path.exists(config_filepath):
-      raise FileNotFoundError(f"{config_filepath} does not exist.")
+    for config in configs:
+      self._config |= config
 
-    with open(config_filepath, "r", encoding="utf-8") as f:
-      self._json = json.loads(f.read())
+    if not self._config:
+      raise FileNotFoundError("Configuration not found.")
 
-    return self
+  @staticmethod
+  def boolify_string_option(value: str):
+    if not value:
+      return None
+    if value.lower().strip() == "true":
+      return True
+    if value.lower().strip() == "false":
```

Comment:
I reckon that any other value should `return False` so you can remove this final conditional